lib/rust/metamodel/src/meta/sexp.rs

```diff
@@ -0,0 +1,181 @@
+//! Producing S-expression representations of data based on reflection information.
```

Comment:
I think I would name this file `s_expr` to be more obvious. Do you like this idea?